srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -73,6 +74,18 @@ def configure_instance(
         )
         disk_size_gb = runner.vm_type.config["disk_size_gb"]
         disk_type = f"projects/{self.project_id}/zones/{self.zone}/diskTypes/pd-ssd"
+
+        automatic_restart = True
+        provisioning_model = "STANDARD"
+        instance_termination_action = "STOP"
+        preemptible = False
+        if "preemptible" in runner.vm_type.config:
+            preemptible = bool(runner.vm_type.config["spot"])
+            if preemptible:
+                provisioning_model = "SPOT"
+                automatic_restart = False
```

Comment:
> Then you check if the key preemptible is set on this schema, but according to the schema preemptible is not defined

For this, it was a mistake! A change all `preemptible` variable name to `spot` and did not test again the project before pushing. Nos it's corrected!

> Then if preemptible is set you retrieve spot

It's just easier this way I think. Because `preemptible` is bool and `provisioning_model = "SPOT"` is not. For me it's more for the user experience. Easier to juste have to set `spot: true` in the conf file that something else.

> Not to mention that all this logic is not required if we were following what I previously said in this [conversation](https://github.com/scality/runner-manager/pull/252/files/ecd443df8bd0c668c4f2dbb796de1d16983676d2#r1221147629)

I know, but a lot changes from that day and I don't agree with this anymore. We can discuss about it on Monday, I'm really open to discussion but I think that `preemptible` is less understandable than `spot`. Even if in the code it does more sens to use `preemptible`.

> That being said I understand that the complexity is then added upon the user configuring the pool of runners. So I'll leave the choice up to you.

I'm really down to take 5 minutes to discuss about it on Monday Standup (either 5 min before or 5 min after it)

Reply:
All your points LGTM and you have addressed the issue in the logic, but I still expect the following requested change:
> Setup a default value for spot or all variables that are non required using the [previously shared method](https://github.com/scality/runner-manager/pull/252/files/ecd443df8bd0c668c4f2dbb796de1d16983676d2#r1221169524). This will allow the code to stop checking if a key was defined.

Reply:
Done! Everything's changed. Can we discuss this tomorrow? I'm not convinced by all this approach